src/app/utilities/api-clients/collections.js

```diff
@@ -30,6 +30,27 @@ export default class collections {
         return http.put(`/zebedee/collection/${collectionID}`, body);
     }
 
+    static getContent(collectionID) {
+        return http.get(`/zebedee/data/${collectionID}?uri=/`);
+    }
+
+    static getContentCollectionDetails(collectionID) {
+        return http.get(`/zebedee/collection/${collectionID}`);
+    }
+
+    static submitPageContentForReview(collectionID, contentURI) {
```

Comment:
We already have methods called `setDatasetStatusToComplete`, `setDatasetStatusToReviewed`, `setDatasetVersionStatusToComplete` and `setDatasetVersionStatusToReviewed`. Could we be consistent? Perhaps zsetContentStatusToComplete` or change dataset methods names?

Reply:
Sounds fair - have updated